app/js/filterFunctions.js

```diff
@@ -113,15 +113,15 @@ function percentageFilter (resultArray){
  */
 function testAllocatedFilter(resultArray) {
     let filterTestAllocated = document.getElementById('testAllocated')
-    let newResultArray = resultArray
-    let chosenTestAllocated = filterTestAllocated.value
+    let newResultArray = resultArray;
+    let chosenTestAllocated = filterTestAllocated.value;
     if (chosenTestAllocated) {
-        newResultArray = []
+        newResultArray = [];
         resultArray.forEach((data)=>{
-            if(data.testAllocated == chosenTestAllocated) {
+            if(data.testId === chosenTestAllocated) {
```

Comment:
why has this changed?

Reply:
I did this so I could re use the handlebars for filtering tests on the editTests page. This had to be done because before the options stored the test names as the value. Now they stored the testId and hence the compare statement had to compare the test ids and not the test names.